cybergis_compute_client/CyberGISCompute.py

```diff
@@ -212,6 +218,25 @@ def create_job(self, maintainer='community_contribution', hpc=None, hpcUsername=
         self.login()
         return Job(maintainer=maintainer, hpc=hpc, id=None, hpcUsername=hpcUsername, hpcPassword=hpcPassword, client=self.client, isJupyter=self.isJupyter, jupyterhubApiToken=self.jupyterhubApiToken, printJob=verbose)
 
+    def run_job_using_params(self,
+                             input_params=[],
+                             maintainer='community_contribution',
+                             hpc="keeling_community",
+                             hpcUsername=None,
+                             hpcPassword=None,
+                             localExecutableFolder={"type": "git",
+                                                    "gitId": "hello_world"},
+                             localDataFolder=None,
+                             localResultFolder=None,
+                             env=None,
+                             slurm=None,
+                             verbose=True):
+        for params in input_params:
+            param_acc = ParamAccumulator(params)
+            job = self.create_job(maintainer, hpc, hpcUsername, hpcPassword)
+            job.set(localExecutableFolder, localDataFolder, localResultFolder, param_acc.params, env, slurm)
+            job.submit()
```

Comment:
This makes sense and works (tested it with a couple of jobs), but we may also need to report the status of jobs. I imagine the way this is going to be used, end-users will want to know that all of their jobs ran and didn't throw errors. Maybe next week we can sit down with Drew and get feedback on that.